src/themes/default/index.scss

```diff
@@ -97,12 +97,23 @@
       color: $highlight1;
     }
 
-    .speed-toggle {
```

Comment:
I checked the branch with two themes, and it did not changed in these two. Are we sure, that this will not brake any custom theme?